### PR TITLE
✨ Nit: Correct a typo in the docstring

### PR DESCRIPTION
**Pull request by shafeeqes**
Nit: Correct a typo in the docstring

### CODE REVIEW

File: pkg/client/fake/client.go [hunk 1]
```diff
@@ -110,6 +112,7 @@ func NewClientBuilder() *ClientBuilder {
 // ClientBuilder builds a fake client.
 type ClientBuilder struct {
 	scheme                *runtime.Scheme
+	clock                 clock.Clock
```
Comment (alvaroaleman) [inline pkg/client/fake/client.go:115]: I don't like this idea. There might be more things in the future where a timestamp is needed and the clock ends up being shared by all of them.

Please use `WithInterceptor` if the hardcoded default result doesn't work for you.

Comment (alvaroaleman) [reply pkg/client/fake/client.go:115]: /hold

Comment (shafeeqes) [reply pkg/client/fake/client.go:115]: Thanks, `WithInterceptor` does suit my needs.
 
> There might be more things in the future where a timestamp is needed and the clock ends up being shared by all of them.

For testing purposes, shouldn't this be okay?

Comment (alvaroaleman) [reply pkg/client/fake/client.go:115]: > For testing purposes, shouldn't this be okay?

You are adding something that is global to the fakeclient but only used in this one place. The next person find another place where they would want to use it and the third person will then want this to be different between the first and the second place and they are stuck.

IMHO we should avoid that altogether by not making this directly configurable at all and redirect people to use the interceptor if they need something more specific than the hardcoded default.

Comment (shafeeqes) [reply pkg/client/fake/client.go]: Okay, I removed the last commit.

**Review by alvaroaleman — APPROVED** [review]

**Pull request merged**